intrahospital_api/management/commands/check_merged_patients.py

```diff
@@ -0,0 +1,232 @@
+"""
+Checks that our Merged MRN cron job is running.
```

Comment:
That's not strictly what it does? 
that would look like `ps aux | grep merge_patients_since` ?